charts/enterprise-logs/templates/compactor-statefulset.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: apps/v1
```

Comment:
Yeah I do think running the compactor as a StatefulSet is preferable semantically. Since the loki-distributed chart uses a Deployment, I would say we should just inherit that for now and consider introducing a change upstream.